src/main.js

```diff
@@ -259,12 +259,11 @@ class MiniGraphCard extends LitElement {
   }
 
   renderStates() {
-    const [firstEntityConfig] = this.config.entities;
     if (this.config.show.state)
       return html`
         <div class="states flex" loc=${this.config.align_state}>
-          ${this.renderState(firstEntityConfig, 0)}
-          <div class="states--secondary">${this.config.entities.map((entityConfig, i) => i > 0 && this.renderState(entityConfig, i) || '')}</div>
+          ${this.renderState(0)}
+          <div class="states--secondary">${i => i > 0 && this.renderState(i) || ''}</div>
```

Comment:
I think, this lost something: We should be iterating over all `this.config.entities` and only rendering the state, if `i > 0` (because we have rendered the first state already).